api/cmd/portainer/main.go

```diff
@@ -618,7 +618,8 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-		adminPasswordHash, err = cryptoService.Hash(string(content))
+		clearPassword := strings.TrimSuffix(string(content), "\n")
+		adminPasswordHash, err = cryptoService.Hash(clearPassword)
```

Comment:
Simply go for:

```
adminPasswordHash, err = cryptoService.Hash(strings.TrimSuffix(string(content), "\n"))
```

Reply:
Ok. Done.
I have also rebased my branch with develop.